docs/cli/overview.md

```diff
@@ -1,23 +1,40 @@
 # Overview
 
-Walking side by side with the Python client, the Inductiva CLI provides 
-a quick and simple interface to interact and perform the most common tasks from
-the terminal. The CLI allows users to quickly manage their tasks, resources and/or
-storage, while their simulations are still running remotely.
+The Inductiva Command Line Interface (CLI) offers a streamlined and efficient way 
+to interact with our platform directly from your terminal. Designed for ease of use, 
+the CLI enables you to manage simulations, resources, and data in real-time, even 
+while your simulations run in the cloud.
 
-The CLI beauty is that its functionalities are a terminal command distance away.
-By doing,
+The power of the Inductiva CLI lies in its simplicity —just a command away: 
 ```bash
 inductiva
 ```
+Now, you can choose from a list of available commands to run via the Inductiva (CLI) to manage your projects and interact with our API:
 
-you will obtain an overview of the available commands and their descriptions.
+````{eval-rst}
```

Comment:
another of our test `sphinx-tabs` @IvanPombo

Reply:
What is this doing? I don't see anything rendered on the final doc:

![Uploading Screenshot 2024-02-21 at 10.21.36.png…]()

Reply:
Maybe we need to add an extension, no @maiyahershey?

Reply:
Yes exactly! I just don't know how to configure this one. We might need to add the extension to the config file so this renders properly. 
@IvanPombo @sergiomsantos

Reply:
Add this: "sphinx_tabs.tabs" to the extensions in the conf.py file.

Reply:
<img width="829" alt="Screenshot 2024-02-21 at 11 13 47" src="https://github.com/inductiva/inductiva/assets/104431973/9479c080-2a21-4233-af28-cf9e561b9453">

Reply:
otimoooo

Reply:
This is the looks now, I think you can now try to manage the coloring based on the tutorials of sphinx-tabs.